File: 1_main.c
//202511716132
//2792131611@qq.com
//陈俊东
#include <stdio.h>
int main() 
{
    float celsius, fahrenheit;
    //获取用户输入的摄氏度
    scanf ("%f", &celsius);
    //进行温度转换
    fahrenheit =celsius * 9.0/5.0+32;
    //输出华氏温度，保留一位小数
    printf ("%.1f\n",fahrenheit);
    return 0;
}

File: 1_main.py
\\202511716132
\\2792131611@qq.com
\\MY_URL
int main() {
    int s;
    scanf ("%d",&s);
    if (s<0 || s>=100){
        printf("成绩错误\n");
}else if (s>=90 && s<=100) {
    printf("A\n");
}else if (s >= 80 && s<=89){
    printf("B\n");
}else if (s >= 70 && s<=79){
}else if (s >= 60 && s<=69){
    printf("D\n");
}else {
    printf("E\n");
}
    return 0;
}
